compiler/natives/src/net/route/zsys_freebsd.go

```diff
@@ -0,0 +1,119 @@
+// Created by cgo -godefs - DO NOT EDIT
+// cgo -godefs defs_freebsd.go
```

Comment:
Let's get rid of this, since it's potentially confusing, and maybe just write a comment like:

```
// This is a copy of https://raw.githubusercontent.com/golang/go/go1.7.1/src/vendor/golang_org/x/net/route/zsys_freebsd_amd64.go.
```

This is so that it'll be easier to remember/figure out where this file came from in the future.